packages/cozy-konnector-libs/src/libs/hydrateAndFilter.spec.js

```diff
@@ -26,10 +26,12 @@ describe('hydrate and filter', () => {
   let entries, filtered
   beforeEach(async () => {
     cozy.data.query.mockReturnValue(
-      asyncResolve([
-        { _id: 1, name: 'Marge', _rev: 2 },
-        { _id: 2, name: 'Homer', _rev: 3 }
-      ])
+      asyncResolve({
+        docs: [
+          { _id: 1, name: 'Marge', _rev: 2 },
+          { _id: 2, name: 'Homer', _rev: 3 }
+        ]
+      })
```

Comment:
How does this test cover the pagination?

Reply:
This test does not cover the pagination, it is just a fix to allow this test to work